### ISSUE TITLE
Cannot build: YELP_HELP_INIT: command not found

### ISSUE THREAD

**Issue by dlazerka** [linuxmint/xviewer#83]
```
 * Xviewer version 2.2.0
```

**Issue**
Cannot build from sources

**Steps to reproduce**
Tried to follow INSTALL guide: there's no `.configure` script, so generated it with `autoconf`:
```
$ autoconf
configure.ac:32: warning: AC_PROG_CC was called before AX_CHECK_ENABLE_DEBUG
aclocal.m4:69: AX_CHECK_ENABLE_DEBUG is expanded from...
aclocal.m4:610: GNOME_DEBUG_CHECK is expanded from...
configure.ac:32: the top level
```

However, running it wasn't successful:
```
...
checking for XML::Parser... ok
./configure: line 15189: YELP_HELP_INIT: command not found
./configure: line 15217: syntax error near unexpected token `0.9.3'
./configure: line 15217: `GOBJECT_INTROSPECTION_CHECK(0.9.3)'
``` 

**Expected behaviour**
No errors and some build artifact to install.

**Other information**
Release 2.2.0

**Comment by leigh123linux** [comment]
Your inability to compile the code isn't our problem, try using some forum instead!

```
./configure: line 15189: YELP_HELP_INIT: command not found
```

The error is in plain English!, INSTALL YELP!!!
